OpenRA.Game/Traits/BodyOrientation.cs

```diff
@@ -62,7 +63,7 @@ public BodyOrientation(ActorInitializer init, BodyOrientationInfo info)
 				if (info.QuantizedFacings >= 0)
 					return info.QuantizedFacings;
 
-				var qboi = self.Info.Traits.GetOrDefault<IQuantizeBodyOrientationInfo>();
+				var qboi = self.Info.Traits.WithInterface<IQuantizeBodyOrientationInfo>().FirstOrDefault();
```

Comment:
In what situation would this be a problem and why would `RenderSprites`/`RenderVoxels` deal with the facings?